commands/upgrade.php

```diff
@@ -25,38 +26,41 @@
 			$version = $release->latest;
 		}
 
+		// checks if the current kirby version is the same as the new one
+		if (version_compare($version, $kirby->version(), '==') === true) {
+			$cli->success('Your Kirby installation is already up to date');
+			exit;
+		}
+
 		// checks current kirby version whether same or higher
 		if (version_compare($version, $kirby->version(), '<=') === true) {
 			throw new Exception('Current Kirby version is the same or higher than the version you are trying to upgrade to');
```

Comment:
@bastianallgeier You added up-to-date message. So wee need to change this compare operator and message.

Reply:
I already added the comparing operator. Not sure I understand what you mean.

Reply:
If kirby version up-to-date (above new your lines), this checks should be like following. For ex case; current version 4.1.1 but version argument 4.0.0.

````php
// checks current kirby version and prevents downgrade
if (version_compare($kirby->version(), $version, '>') === true) {
    throw new Exception('Current Kirby version is higher than the version you are trying to upgrade to');
}
````

Reply:
Ah, now I get it. I fixed it on develop

Reply:
Thank you. Also updating message would be great. Because can't be same version since your new check lines 🙈

Reply:
### Current message
Current Kirby version is the same or higher than the version you are trying to upgrade to

### Possible
Current Kirby version is higher than the version you are trying to upgrade to

Reply:
Done in develop branch https://github.com/getkirby/cli/commit/b86089f9d4b42119a883a029bd1767b33ef1b185

Reply:
You are right! I made a small adjustment to your text. Now it looks good to me.